components/raftstore-v2/src/batch/store.rs

```diff
@@ -373,10 +374,21 @@ impl<EK: KvEngine, ER: RaftEngine> StoreSystem<EK, ER> {
         workers
             .store_writers
             .spawn(store_id, raft_engine.clone(), None, router, &trans, &cfg)?;
-        let read_scheduler = workers.async_read_worker.start(
-            "async-read-worker",
-            ReadRunner::new(router.clone(), raft_engine.clone()),
+
+        let snap_mgr = SnapManagerBuilder::default().build(
```

Comment:
Why do we need a manager?

Reply:
yes, here still use snap manager to register the `snap_entry`, and do some statistics or state check same as old way.

Reply:
All these can be done without a manager. We don't even need to register it.

Reply:
> All these can be done without a manager. We don't even need to register it.

What's the benefit of doing it without a manager? Is there any key difference of snapshot management between v1 and v2? @BusyJay

Reply:
PTAL @BusyJay

Reply:
We don't need GC anymore, just delete the files when unused. Generating a snapshot is not as expansive as before.